Mon Oct  2 15:52:52 ART 2006  Ricardo Markiewicz <rmarkie@fi.uba.ar>
  * Install missing icon.
  Also, use the SVG version of the icon.
diff -rN -u old-oregano-stable/SConstruct new-oregano-stable/SConstruct
--- old-oregano-stable/SConstruct	2006-10-02 15:53:55.000000000 -0300
+++ new-oregano-stable/SConstruct	2006-10-02 15:53:55.000000000 -0300
@@ -115,8 +115,9 @@
 SConscript ('po/SConscript');
 
 # Install Target #
-CEnv.Alias ('install', CEnv.Command ('oregano.keys', 'oregano.keys.in', "sed 's/@icondir@/"+os.path.join (CEnv['PREFIX'], CEnv['DATADIR'],"mine-info").replace ("/", "\\/")+"/' < $SOURCE > $TARGET"))
+CEnv.Alias ('install', CEnv.Command ('oregano.keys', 'oregano.keys.in', "sed 's/@icondir@/"+os.path.join (CEnv['DATADIR'], "pixmaps").replace ("/", "\\/")+"/' < $SOURCE > $TARGET"))
 CEnv.Alias ('install', CEnv.Command ('oregano.xml', 'oregano.xml.in', "cp $SOURCE $TARGET"))
+CEnv.Alias ('install', CEnv.Install (os.path.join (CEnv['INSTALL_DIR'], 'share/pixmaps'), Split('gnome-oregano.svg')))
 CEnv.Alias ('install', CEnv.Install (os.path.join (CEnv['INSTALL_DIR'], 'share/mime/packages'), Split('oregano.xml')))
 CEnv.Alias ('install', CEnv.Install (os.path.join (CEnv['INSTALL_DIR'], 'share/mime-info'), Split('oregano.mime oregano.keys')))
 CEnv.Alias ('install', CEnv.Install (os.path.join (CEnv['INSTALL_DIR'], 'share/applications'), Split('oregano.desktop')))
diff -rN -u old-oregano-stable/oregano.keys.in new-oregano-stable/oregano.keys.in
--- old-oregano-stable/oregano.keys.in	2006-10-02 15:53:55.000000000 -0300
+++ new-oregano-stable/oregano.keys.in	2006-10-02 15:53:55.000000000 -0300
@@ -1,4 +1,4 @@
 application/x-oregano:
 	open=oregano %f
 	view=oregano %f
-	icon-filename=@icondir@/gnome-oregano.png
+	icon-filename=@icondir@/gnome-oregano.svg

